elixir/apps/api/lib/api/gateway/views/client.ex

```diff
@@ -7,7 +7,9 @@ defmodule API.Gateway.Views.Client do
       public_key: client.public_key,
       preshared_key: preshared_key,
       ipv4: client.ipv4,
-      ipv6: client.ipv6
+      ipv6: client.ipv6,
+      version: client.last_seen_version,
+      device_serial: client.device_serial
```

Comment:
Unfortunately we don't get device serial on every device. It would good to give the admin whatever we have here (and follow up with documentation when we ship this feature).


We'll probably want to omit these fields entirely from the response if they're nil right? You could use `%{...} |> Enum.reject(fn {_k, v} -> is_nil(v) end) |> Enum.into(%{})` for that.

```suggestion
      device_uuid: client.device_uuid,
      firebase_installation_id: client.firebase_installation_id,
      identifier_for_vendor: client.identifier_for_vendor,
      device_serial: client.device_serial
```